### ISSUE TITLE
Create a new page for #2PlaysAMonth initiative

### ISSUE THREAD

**Issue by atapas** [reactplay/activities#109]
**Is your feature request related to a problem? Please describe.**

We need to create a new initiative page for the event #2PlaysAMonth initiative.

The Motto: 
- Create 2 plays in the month. 
- At the end of the month, we judge the submissions based on criteria. 
- We publish 3 Winners and 3 Special Mentions. The initiative may have sponsors. In that case, Winners will get prizes.

**Describe the solution you'd like**
Right now the home page route sets to: `hackrplay/2022/home`. Create a new route called `2playsamonth/2023/home` and the new home should have the followings:

- Use the exact structure like this page: https://hustles.reactplay.io/hackrplay/2022/home (Hack-R-Play)
- We will not show the WINNERS link in the header and the WINNERS section in the beginning.
- We will change all required data that is related to our new initiative(Contact @atapas for data)
- We need to change the logo and images(@nirmalkc will give that)
- We need to change the color and font for the theme(@nirmalkc will do that).
- If we have a sponsor, will add the sponsor section, else remove it.

**Additional context**
Once this page is up and deployed, we need to link it from ReactPlay main page from a banner.

- Please use this branch `https://github.com/reactplay/activities/tree/2-plays-a-month`

@atapas will add all updates here in this issue.

**Comment by Amyx000** [comment]
Hey @atapas , so 2 plays a month home structure will be same as hackerplay section like about, judge, faq, winner

**Comment by atapas** [comment]
> Hey @atapas , so 2 plays a month home structure will be same as hackerplay section like about, judge, faq, winner

You are right.

**Comment by Amyx000** [comment]
Should I start with basic templating of the page with dummy data?

**Comment by atapas** [comment]
> Should I start with basic templating of the page with dummy data?

Sure but use existing components as much as possible.

**Comment by Amyx000** [comment]
yup!

**Comment by Amyx000** [comment]
The page is setup, just copied the home from 2022, the changes can be done accordingly

<img width="823" alt="image" src="https://user-images.githubusercontent.com/104687128/213861636-1efbc5bd-6c56-416d-8074-55f0873423d1.png">

**Comment by debjit** [comment]
Ok I will setup and check the code too.

**Comment by atapas** [comment]
> The page is setup, just copied the home from 2022, the changes can be done accordingly
> 
> 
> 
> <img width="823" alt="image" src="https://user-images.githubusercontent.com/104687128/213861636-1efbc5bd-6c56-416d-8074-55f0873423d1.png">

Hide the Winner section and Winner link

**Comment by atapas** [comment]
@Amyx000 & @debjit 

Here are the changes:

1. Replace the following section text:
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/3633137/213897950-06232ae3-47da-43c7-bb92-68e791b4b1d2.png">

with these text:

`ReactPlay brings you an opportunity to participate in the month-long drive to learn and contribute to Open Source. Join the #2PlaysAMonth and build two projects(plays) in the month of February. You will learn from expert code reviews while contributing to Open Source - you may also win some exciting prizes.`

2. About Section(https://hustles.reactplay.io/hackrplay/2022/home#about)

Leave this part as is: 
<img width="424" alt="image" src="https://user-images.githubusercontent.com/3633137/213898085-c9bb42d7-b4b7-4c87-a17f-b8374cd16aeb.png">

For this section,
<img width="521" alt="image" src="https://user-images.githubusercontent.com/3633137/213898095-39f6d213-e280-4f20-88ac-b163d3d2a6e2.png">

here is the content:

```
#2PlaysAMonth is an initiative from the RaectPlay community to help you learn, build, and simultaneously contribute to open source throughout the month of February.

Would you be up for it? Here are a few things to keep in mind before you get started:

- You have to complete two plays(React Projects) in ReactPlay between February 1st to February 28th 2023. By complete, we mean the play Pull Requests must be merged by the community reviewers.
- You must follow the [Create a Play](https://github.com/reactplay/react-play/blob/main/CREATE-PLAY.md) steps to create the play.
- Your play must have at least `#2plays1month` tag added to it while creating.
- You can build any play of your choice. If you are looking for ideas, [here are some you can get inspired by](https://reactplay.io/ideas).
- You must share a weekly Tweet and/or LinkedIn Post about your journey with the hashtag `#2plays1month` and tag ReactPlay's handle(@ReactPlayIO)

Please join the [ReactPlay discord server](https://discord.gg/UfYj4MvW9A) to get any help while you are building.
```
There will not be any `Read More` as of now.

3. About this section

<img width="786" alt="image" src="https://user-images.githubusercontent.com/3633137/213898706-8f9bd193-3cb5-47a1-a07f-168ed85a9afd.png">

Use these 3 points:
- Create 2 Plays using React
- Share your journey publicly
- Get the plays reviewed and merged


4. For the judges section, just have @koustov and @atapas for now. Remove other two.

5. For the Sponsors section, keep the `Our Sponsors` heading. Remove Nhost logo and text. We will add things here later.

6. For this section

<img width="1376" alt="image" src="https://user-images.githubusercontent.com/3633137/213898852-d1a1dbe2-9724-46a8-af8d-6cb1f0d27d8c.png">

Use this:
- Keep the heading an image as is
- The text change to: `Learning is a journey than a destination. We, developers, need avenues, motivations, and opportunities to keep going. Join the #2PlayAMonth initiative to experience it. It will allow you to build a ReactJS app using and learning from code reviews. Why waiting? Get started today.`

7. FAQ I will add a separate comment sometime later today.

**Comment by atapas** [comment]
The FAQ section @debjit and @Amyx000 

Q: Why should I participate in the #2PlaysAMonth event?
A: Do you like learning while building something? How about a platform that allows you to build a ReactJS application end-to-end, learn from the review comments, contribute to open source, showcase it to the world, and help build networking? #2PlaysAMonth aims precisely the same. Join the drive brought to you by ReactPlay and start building cool projects that we can't wait to see!

Q: Do I need to Register for #2PlaysAMonth?
A: Nope. You can create the play by following the criteria mentioned in this page's About(link it) section.

Q: Is it mandatory to create 2 plays?
A: Yes. That's one of the qualifying criteria. You must create and complete 2 plays.

Q: What if my play review gets delayed?
A: You must ensure you give the reviewers enough time to review your code and provide comments. We're looking forward to 4-5 days(based on how actively you respond to the comments) to complete the review process of a play. Try submitting your plays at least 7-10 days before the end date to avoid messy situations. If your review gets delayed by reviewers, we will ensure you the extra time, but that will be a rare case!

Q: Can we participate as a team?
A: Nope. It's an individual event.

Q: Can I use anything other than React?
A: ReactPlay is a platform to help web developers learn ReactJs and build projects. For this initiative, RaectJs is a must.

Q: Can I submit my old React project as a play?
A: You can. However, could you please mention the old project in the play description? Also, the play should be something other than a 1-1 matching your existing project. You must bring some changes/ideas into it.

Q: Can I submit code/project of someone else?
A: You shouldn't. Be authentic and ethical. It's an event to learn, not alone to win.

Q: Is creating an issue for my PR a must?
A: Yes. We will disqualify the play submission that doesn't have an associated issue.

Q: Is sharing my journey on Twitter and LinkedIn of the #2PlaysAMonth a must?
A: Yes, we encourage you to build and learn publicly. Make sure you add the tag #2playsamonth and ReactPlay handle when you post them on social media.

Q: When and How the #2PlaysAMonth result will be published?
A: The result will be published on 15th March. We will publish it on our website, blog, and Twitter handle(@reactplayio).

Q: Do I get prizes?
A: We will publish 3 winners and 3 special mentions based on our judging criteria. All winners, special mentions, and participants who completed the target will receive digital badges. We are working with our sponsors to decide the prizes for the winners.

Q: I have questions. Where can I ask them?
A: Got a question? Please join our Discord(link it). You can ask us anything in the hack-r-play channel.

**Comment by Amyx000** [comment]
> 

hey @atapas to remove the winner from navbar, right now the navbar has the static data which from layout component, can I change it and make it dynamic by taking value from props so it can be used in any homepage by providing data for navbar?
<img width="420" alt="image" src="https://user-images.githubusercontent.com/104687128/213900367-7adfb7d0-163a-4416-a55d-f576f515e8a5.png">

I can take home links from props so that I can remove the winner

**Comment by atapas** [comment]
> > 
> 
> 
> 
> hey @atapas to remove the winner from navbar, right now the navbar has the static data which from layout component, can I change it and make it dynamic by taking value from props so it can be used in any homepage by providing data for navbar?
> 
> <img width="420" alt="image" src="https://user-images.githubusercontent.com/104687128/213900367-7adfb7d0-163a-4416-a55d-f576f515e8a5.png">
> 
> 
> 
> I can take home links from props so that I can remove the winner
> 
> 

Yes, feel free to do any such improvements 👍

**Comment by Amyx000** [comment]
Ok👍 and to start working together someone has to push the 2023 folder with home.js first otherwise it will be conflict, should I just push the page first?

**Comment by atapas** [comment]
> Ok👍 and to start working together someone has to push the 2023 folder with home.js first otherwise it will be conflict, should I just push the page first?

Yes, let me merge that first

**Comment by koustov** [comment]
> 

Additionally,
- One section needs to be created on the home page for `Archives`. A link should be provided to reach `2022/home` page
- All hustle events need to be tied up with a specific ID. F2 2PlayMonth, the ID is `26839655-eaeb-44c3-927e-f334cab413b8
`. The only place where this ID would be required is "Winners" section.  Winners section can be created now however it shouldn't show up if there are no winners entering in `hackathon/winners` table

**Comment by atapas** [comment]
> > 
> 
> 
> 
> Additionally,
> 
> - One section needs to be created on the home page for `Archives`. A link should be provided to reach `2022/home` page
> 
> - All hustle events need to be tied up with a specific ID. F2 2PlayMonth, the ID is `26839655-eaeb-44c3-927e-f334cab413b8
> 
> `. The only place where this ID would be required is "Winners" section.  Winners section can be created now however it shouldn't show up if there are no winners entering in `hackathon/winners` table

Right. Let's take that as a separate issue @koustov . Can you please create one? We can work on that independently to this one.

**Comment by koustov** [comment]
- `Submission` link should redirect to reactplay/plays with prefilled filter of `2PlaysAMonth tag

**Comment by koustov** [comment]
> > > 
> > 
> > 
> > Additionally,
> > 
> > * One section needs to be created on the home page for `Archives`. A link should be provided to reach `2022/home` page
> > * All hustle events need to be tied up with a specific ID. F2 2PlayMonth, the ID is `26839655-eaeb-44c3-927e-f334cab413b8
> > 
> > `. The only place where this ID would be required is "Winners" section.  Winners section can be created now however it shouldn't show up if there are no winners entering in `hackathon/winners` table
> 
> Right. Let's take that as a separate issue @koustov . Can you please create one? We can work on that independently to this one.

Yeap, the winners one can be taken separately but lets include the archive one here

**Comment by atapas** [comment]
> > > > 
> 
> > > 
> 
> > > 
> 
> > > Additionally,
> 
> > > 
> 
> > > * One section needs to be created on the home page for `Archives`. A link should be provided to reach `2022/home` page
> 
> > > * All hustle events need to be tied up with a specific ID. F2 2PlayMonth, the ID is `26839655-eaeb-44c3-927e-f334cab413b8
> 
> > > 
> 
> > > `. The only place where this ID would be required is "Winners" section.  Winners section can be created now however it shouldn't show up if there are no winners entering in `hackathon/winners` table
> 
> > 
> 
> > Right. Let's take that as a separate issue @koustov . Can you please create one? We can work on that independently to this one.
> 
> 
> 
> Yeap, the winners one can be taken separately but lets include the archive one here

We don't have the archive page design.. so not keen on blocking this PR work. We can take archive also as separate issue so that we can ask someone to start with it in parallel. What say?

**Comment by debjit** [comment]
> 

I think we should create an dedicated index page later and create separate page template for hackathons.

**Comment by koustov** [comment]
ok, Lets push it some other issue however I was suggesting to have archive section pretty much simple in nature, not archive page

**Comment by Amyx000** [comment]
@atapas I made PR for home page, so that @debjit can start working on it

**Comment by atapas** [comment]
> @atapas I made PR for home page, so that @debjit can start working on it

PR link?

**Comment by Amyx000** [comment]
#111

**Comment by atapas** [comment]
> #111 

Merged. Now you both can distribute and do. Make sure to create all new branches from the 2-plays-a-month branch.

**Comment by atapas** [comment]
> ok, Lets push it some other issue however I was suggesting to have archive section pretty much simple in nature, not archive page

@koustov can you plz create one with expectations? We can assign to a contributor.

**Comment by Amyx000** [comment]
@debjit start editing the 2playsamonth/2023/home.js file

**Comment by debjit** [comment]
OK sure!

**Comment by atapas** [comment]
> > ok, Lets push it some other issue however I was suggesting to have archive section pretty much simple in nature, not archive page
> 
> @koustov can you plz create one with expectations? We can assign to a contributor.

I think we have an issue already: https://github.com/reactplay/activities/issues/105

**Comment by debjit** [comment]
[PR Requestr](https://github.com/reactplay/activities/pull/112)
![ReactPlay-presents-HACK-R-PLAY](https://user-images.githubusercontent.com/565922/213926532-4836ca3e-9227-4f7d-9a1d-b09b6888b973.png)

**Comment by nirmalkc** [comment]
### Design #1

![image](https://user-images.githubusercontent.com/6359059/213972438-fbd68bb3-1308-48c5-9f68-29b12c686120.png)

**Comment by atapas** [comment]
> ### Design #1
> ![image](https://user-images.githubusercontent.com/6359059/213972438-fbd68bb3-1308-48c5-9f68-29b12c686120.png)

All good @nirmalkc 

Can we just change the image to something else?

<img width="571" alt="image" src="https://user-images.githubusercontent.com/3633137/213974207-3361ad60-d138-42f9-b933-d5264ad0e737.png">


@debjit @Amyx000, you don't have to take care of these styling changes now. Once your work is merged, @nirmalkc will do it directly. Thanks!

**Comment by nirmalkc** [comment]
### Design #2
![image](https://user-images.githubusercontent.com/6359059/213976891-9ef9fa27-ff5d-4ddf-af33-baa612b70604.png)

**Comment by nirmalkc** [comment]
@atapas Here is the link that has the detailed design specs like colors, logo etc.,
https://xd.adobe.com/view/7ba992bb-560f-4298-b149-300f401b6d2a-2efa/

Please start with the development work.

**Comment by atapas** [comment]
It has been deployed. There is one issue with the OG image that we will fix as part of the next issue.
